conf/locale/locale_en-US.ini

```diff
@@ -485,8 +485,7 @@ issues.commit_ref_at = `referenced this issue from a commit <a id="%[1]s" href="
 issues.poster = Poster
 issues.collaborator = Collaborator
 issues.owner = Owner
-issues.sign_up_for_free = Sign up for free
-issues.sign_in_require_desc = to join this conversation. Already have an account? <a href="%s">Sign in to comment</a>
+issues.sign_in_require_desc = <a href="%s">Sign in to comment</a>
```

Comment:
Agreed, but as usual, I try to keep my PR focused on a topic
(hiding the "Sign up for free" part, in this case).

I'm leaving for holiday so won't be able to improve this PR
any soon.